### ISSUE TITLE
GameStoppingServerEvent not firing

### ISSUE THREAD

**Issue by TypicalDarkness** [SpongePowered/SpongeForge#471]
For some reason this doesn't fire:
```java
@Listener void onServerStop(GameStoppingServerEvent e) {}
```
Tested on 979.

**Comment by TypicalDarkness** [comment]
I guess this is has nothing to do with SpongeForge. I'll open the issue at SpongeAPI

**Comment by gabizou** [comment]
@TypicalDarkness It does have to do with the implementation, not API.

**Comment by JBYoshi** [comment]
I have a PR that fixes this - #425.

**Comment by simon816** [comment]
@JBYoshi that's for `GameStoppingEvent`, this is `GameStoppingServerEvent`

The event works fine
```java
@Listener
public void onServerStop(GameStoppingServerEvent e) {
    System.out.println(e);
}
```
```
[STDOUT]: [com.simon816.eventtest.SpongePlugin:onServerStop:57]: net.minecraftforge.fml.common.event.FMLServerStoppingEvent@3e5d4255
```

@TypicalDarkness Ah, I think I know. You need to make your handler public, the code you've got there is package-private.
